### PR TITLE
add cutom website and duplicate marialunarillos

### PR DESCRIPTION
**Pull request by albertManzano**

### CODE REVIEW

File: staff/albert-manzano/templates/custom/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <ul class="side-nav" >
+                <li class="side-nav__item">
+                    <img class="side-nav__icon side-nav__icon--small side-nav__left-corner side-nav__icons--glow " id="close-nav" src="icons/close.png">
+                </li>
+                <li class="side-nav__item">
+                    <a href="greenhouses.html"><img class="side-nav__icon side-nav__icons--glow " src="icons/aquaponics.png"></a>
+                    Greenhouses
+                </li>
+                <li class="side-nav__item">
+                    <a href="forecast.html"><img class="side-nav__icon side-nav__icons--glow " src="icons/forecast.png"></a>
```
Comment (lichab) [inline staff/albert-manzano/templates/custom/index.html:23]: Small correction here, the modifier `--glow` should be applied to an already existing `side-nav__icon` class. But you are just creating a new class called `side-nav__icons--glow` notice the "s" at the end of icon. Just a small correction regarding to BEM. Will merge the branch anyway, you don't need to modify it here just take it into account for the next one :smile:

File: staff/albert-manzano/templates/custom/style.sass [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    justify-content: end
+    align-content: center
+    margin-top: auto
+    margin-bottom: auto
+    margin-right: 15px
+
+.side-nav
+    display: grid
+    grid-auto-rows: auto
+
```
Comment (lichab) [inline staff/albert-manzano/templates/custom/style.sass:58]: here following the previous comment, you would be able to do this:

```css
.side-nav
    display: grid
    grid-auto-rows: auto

    &__icon
        height: 50px
        width: 50px
        margin-top: 25%
        margin-left: 30px
        cursor: pointer
        border: 0px solid #ffffff'

        &--glow
            transition-duration: 0.3s
            transition-property: box-shadow
            -webkit-tap-highlight-color: rgba(0, 0, 0, 0)
            transform: translateZ(0)
            box-shadow: 0 0 1px rgba(0, 0, 0, 0)

            &:hover
                box-shadow: 0 0 8px rgba(0, 0, 0, 0.6)
                
        &--small
            height: 20px
            width: 20px        
```

**Comment by lichab** [comment]
The comments are just for improvement but you did a great job!

**Comment by albertManzano** [comment]
cheers mate! your comments are exactly what i am looking for!
I will try to put in to practice.
Thanks for the feedback.

Gretting

**Pull request merged**